utils/utils.c

```diff
@@ -332,6 +332,55 @@ gbRunner(char *cmd)
 }
 
 
+char*
+gbRunnerGetOutput(char *cmd)
+{
+  FILE *fp;
+  char *tptr;
+  char *buf = NULL;
+
+
+  LOG("mgmt", GB_LOG_DEBUG, "command, %s", cmd);
+
+  if (GB_ALLOC_N(buf, 1024) < 0) {
+    LOG("mgmt", GB_LOG_ERROR,
+        "gbRunnerGetOutput: error allocating memory (%s)", strerror(errno));
+    return NULL;
+  }
+
+  fp = popen(cmd, "r");
+  if (fp) {
+    size_t newLen = fread(buf, sizeof(char), 1024, fp);
+    if (ferror(fp) != 0) {
+      LOG("mgmt", GB_LOG_ERROR,
+          "reading command output for %s failed (%s)", cmd, strerror(errno));
+      buf[0] = '\0';
+      pclose(fp);
```

Comment:
Please remove this line, we will do it in the end of gbRunnerGetOutput().

Reply:
good catch.